library-viewbinding/src/main/java/com/xwray/groupie/viewbinding/BindableItem.java

```diff
@@ -0,0 +1,74 @@
+package com.xwray.groupie.viewbinding;
+
+import androidx.annotation.NonNull;
+import androidx.viewbinding.ViewBinding;
+
+import android.view.View;
+
+import com.xwray.groupie.Item;
+
+import java.util.List;
+
+/**
+ * The base unit of content for a GroupAdapter.
+ * <p>
+ * Because an Item is a Group of size one, you don't need to use Groups directly if you don't want;
+ * simply mix and match Items and add directly to the adapter.
+ * <p>
+ * If you want to use Groups, because Item extends Group, you can mix and match adding Items and
+ * other Groups directly to the adapter.
+ *
+ * @param <T> The ViewBinding subclass associated with this Item.
+ */
+public abstract class BindableItem<T extends ViewBinding> extends Item<GroupieViewHolder<T>> {
+
+    public BindableItem() {
+        super();
+    }
+
+    protected BindableItem(long id) {
+        super(id);
+    }
+
+    @NonNull
+    protected abstract T initializeViewBinding(@NonNull View view);
```

Comment:
Added an abstract method for providing ViewBinding instance as @lkishor suggested in #322 .